### ISSUE TITLE
`OpenTelemetry metric error occurred` error log

### ISSUE THREAD

**Issue by yanns** [apollographql/router#4047]
**Describe the bug**
In the log, the following event is emitted:
```
OpenTelemetry metric error occurred: Metrics error: Instrument description conflict, using existing. Instrument apollo_router_http_requests_total_total, Existing: "Total number of HTTP requests made.", dropped: ""
```

**To Reproduce**
Running the router

**Expected behavior**
No log

**Desktop (please complete the following information):**
 - Version: 1.32.0

**Comment by andriihrachov** [comment]
experiencing the same

**Comment by BrynCooke** [comment]
It looks like there are multiple points in the codebase where this metric is being incremented, and they don't match.

We're currently refactoring to move this all to the same place, but in the meantime we'll create a fix that will prevent the issues for occuring.

**Comment by abernix** [comment]
This should be fixed by #4065, which will come out in v1.33.1 #4066

**Comment by BrynCooke** [comment]
We believe this is finally squashed in 1.33.2
